airqo-platform/AirQo-frontend
# Join platform analytics

- What does this PR do?
User Login for the Analytics Product
- What JIRA tasks are related to this PR?
[Request access](https://airqoteam.atlassian.net/browse/NLYTCS-1) AND [Log out or time out](https://airqoteam.atlassian.net/browse/NLYTCS-28?atlOrigin=eyJpIjoiZGJmYTJmZTY1NWMzNGU5ZGEyNmZhZmZlMDAwOTBiNDYiLCJwIjoiaiJ9)
- How do I test out this PR?
1. Git clone this repo `cd AirQo-frontend/analytics` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local` to test out the server that runs on port 5000. For windows users, you might want to first switch to a new testing branch and then editing the **package.json** script line from  `"local": "PORT=5000 react-scripts start"`, to  `"local": "react-scripts start"`,

2. Please ensure that you also have the auth service API running in another browser since it is consuming the localhost endpoints. That is: open a new tab of your terminal and: `cd AirQo-api/src/auth-servic`e and then run `npm install`. Finally, run` npm run dev` to get the API server running on port 3000. You will also need to have a .env file in this service for things to run well. Contents of this .env file can be requested for from Martin separately. 
- Which end points should are ready for testing?
None
